documentation/src/main/wiki/TestExecutor.md

```diff
@@ -39,6 +39,25 @@ This endpoint returns JSON object with following fields:
 | `xunitReportUrl` | Link to xUnit report generated by the suite run.    |
 | `errorMessage`   | Contains an error message. Only this field will be present if suite start fails. |
 
+##### Rerun suite
+
+###### Request
+
+* **URL**: `/suite-rerun`
+* **HTTP Method**: POST
+* **Parameters**:
+    * `correlationId` - Correlation ID of the suite to rerun, used to get it from the database
```

Comment:
Would it be possible to also support `company, project, suite, version`?
Does that make sense?
It could be done as an improvement in separate PR. If you think this is valid, please create proper ticket.